### ISSUE TITLE
Unable to download

### ISSUE THREAD

**Issue by estoi** [ArkeshGKalathiya/capacitor-pdf-reader#1]
```bash
npm install netgalaxy-pdf-reader
```

```bash
npm ERR! code E404
npm ERR! 404 Not Found - GET https://repo.huaweicloud.com/repository/npm/netgalaxy-pdf-reader
npm ERR! 404 
npm ERR! 404  'netgalaxy-pdf-reader@*' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by ArkeshGKalathiya** [comment]
Hey @estoi,

It's not linked to NPM repo yet, you can use GitHub link instead

i.e `npm install https://github.com/ArkeshGKalathiya/capacitor-pdf-reader`

**Comment by estoi** [comment]
@ArkeshGKalathiya Is this project still under maintenance?

**Comment by ArkeshGKalathiya** [comment]
Yes, I have recently updated it. It only supports android though. Because for IOS there is another plugin.

**Comment by ArkeshGKalathiya** [comment]
Let me know if that works for you.

**Comment by estoi** [comment]
> 让我知道这是否适合你。

downloading

**Comment by ArkeshGKalathiya** [comment]
This plugin was created to support bookmarks on android, otherwise there is this plugin called cordova-plugin-document-viewer ( for iOS part ), you can use it. Another solution is pdf.js, but at the time it was crashing for large pdfs.

**Comment by estoi** [comment]
Still unable to download

<img width="574" alt="image" src="https://user-images.githubusercontent.com/44252820/156685653-a0a23e46-e7ea-405f-b535-2b3110b42528.png">

**Comment by ArkeshGKalathiya** [comment]
It seems network related issue, can you make sure that you can download other plugins too via GitHub?

i.e 

`npm install https://github.com/sitewaerts/cordova-plugin-document-viewer`

**Comment by estoi** [comment]
> 

There is no problem with this link

**Comment by ArkeshGKalathiya** [comment]
Then I don't know what is the issue, repo is already public. You can download the source folder, and do "npm install ./path/to/plugin
"
